interop-tests/src/test/scala/akka/grpc/interop/GrpcInteropSpec.scala

```diff
@@ -33,14 +60,48 @@ object AkkaHttpServerProviderJava extends AkkaHttpServerProvider {
   })
 }
 
-object AkkaHttpClientProviderScala extends AkkaHttpClientProvider {
+abstract class AkkaClientProviderScala(backend: String) extends AkkaClientProvider {
   val label: String = "akka-grpc scala client tester"
 
-  def client = AkkaGrpcClientScala(settings => implicit sys => new AkkaGrpcScalaClientTester(settings))
+  def client = AkkaGrpcClientScala(settings => implicit sys => new AkkaGrpcScalaClientTester(settings, backend))
 }
 
-object AkkaHttpClientProviderJava extends AkkaHttpClientProvider {
+object AkkaNettyClientProviderScala extends AkkaClientProviderScala("netty") {
+  val pendingCases =
+    Set(
+      "cancel_after_begin",
+      "cancel_after_first_response",
+      "timeout_on_sleeping_server",
+      "custom_metadata",
+      "client_compressed_unary",
+      "client_compressed_streaming",
+      "server_compressed_unary")
+}
+
+object AkkaHttpClientProviderScala extends AkkaClientProviderScala("akka-http") {
+  val pendingCases =
+    Set(
+      "cancel_after_begin",
```

Comment:
They are the same right now because the 3 client implementations now successfully support the same set of interop tests.

I'll move the set to `AkkaClientProviderScala`: when we implement further tests, I think we should strive to implement them for all backends at the same time. If for some reason we can't, then we can split the set back out then.